src/push/request.ts

```diff
@@ -58,6 +58,7 @@ export async function sendReqAndHandleError<T>(
   options: APIRequestOptions
 ): Promise<T> {
   const { statusCode, body } = await sendRequest(options);
+
   return (
     await handleError(statusCode, options.url, body)
```

Comment:
Can we add the chunk size and payload size information on this error? It will be helpful to know how big the payload was and how many monitors per chunk were sent